packages/resource-deployment/templates/dashboard.template.json

```diff
@@ -1014,11 +1014,177 @@
+                                        },
+                                        {
+                                            "name": "TimeRange",
+                                            "isOptional": true
+                                        },
+                                        {
+                                            "name": "Version",
+                                            "value": "1.0"
+                                        },
+                                        {
+                                            "name": "PartId",
+                                            "value": "6fb6698a-49fe-4cca-b565-8fc6b6367b01"
+                                        },
+                                        {
+                                            "name": "PartTitle",
+                                            "value": "E2E Test Groups"
+                                        },
+                                        {
+                                            "name": "PartSubTitle",
+                                            "value": "Results for most recent test run"
+                                        },
+                                        {
+                                            "name": "resourceTypeMode",
+                                            "value": "components"
+                                        },
+                                        {
+                                            "name": "ControlType",
+                                            "value": "AnalyticsGrid"
+                                        },
+                                        {
+                                            "name": "Dimensions",
+                                            "isOptional": true
+                                        },
+                                        {
+                                            "name": "DashboardId",
+                                            "isOptional": true
+                                        },
+                                        {
+                                            "name": "SpecificChart",
+                                            "isOptional": true
+                                        }
+                                    ],
+                                    "type": "Extension/AppInsightsExtension/PartType/AnalyticsPart",
+                                    "settings": {
+                                        "content": {
+                                            "PartTitle": "E2E Test Groups",
+                                            "PartSubTitle": "Results for most recent test runs"
```

Comment:
Can we have list of tests instead of groups or drill down option to see what test(s) is falling